test/gringotts_test.exs

```diff
@@ -36,10 +42,8 @@ defmodule GringottsTest do
     end
   end
 
-  setup_all do
-    Application.put_env(:gringotts, GringottsTest.FakeGateway, [
-          adapter: GringottsTest.FakeGateway,
-          some_auth_info: :merchant_secret_key])
+  setup do
+    Application.put_env(:gringotts, GringottsTest.FakeGateway, @test_config)
     :ok
```

Comment:
why this `:ok`?

Reply:
`setup` and `setup_all` macros will merge the block's return value into the test contexts, if nothing is to be inserted into context, an `:ok` must be returned. [ExUnit docs](https://hexdocs.pm/ex_unit/ExUnit.Callbacks.html#module-context)